### PR TITLE
GET /status/* APIs for Liveness and Readiness Probes

### PR DESCRIPTION
**Pull request by hfuss**
Signed-off-by: hfuss <haydenfuss@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,6 +15,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/hyperledger/firefly-transaction-manager v0.9.10 => github.com/kaleido-io/firefly-transaction-manager v0.0.0-20220902203522-fcba476c9ec5
```
Comment (hfuss) [inline go.mod:18]: Remove once https://github.com/hyperledger/firefly-transaction-manager/pull/33 is merged

**Comment by hfuss** [comment]
Confirmed against Polygon Mumbai the status endpoints work as intended:
```shell
~ $ curl http://mumbai-evmconnect:5000/status
{"up":true}
~ $ curl http://mumbai-evmconnect:5000/status/ready
{"ready":true,"downstreamDetails":{"chainID":"80001"}}
~ $ curl http://mumbai-evmconnect:5000/status/ready
{"ready":true,"downstreamDetails":{"chainID":"80001"}}
```

And that ERC20 connector `v1.1.1` now passes its own readiness probes in K8s (which calls `/status`):
```
[Nest] 19  - 09/03/2022, 12:30:04 AM     LOG [RequestLogging] GET /api/v1/health/liveness
[Nest] 19  - 09/03/2022, 12:30:04 AM     LOG [RequestLogging] GET /api/v1/health/readiness
```

**Review by peterbroadhurst — APPROVED** [review]
Looks great, thanks @hfuss

**Review by nguyer — APPROVED** [review]
And I approve the changes Peter made 🙂

File: go.mod (hunk 1 above)
Comment (nguyer) [inline go.mod:18]: This still needs to be removed

**Review by nguyer — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (peterbroadhurst) [reply go.mod]: Sorry about that - failed to push a local change

**Review by nguyer — APPROVED** [review]
NOW it looks good. Sorry for the waffle.

**Pull request merged**